np.plugin-flow-skeleton/src/helloWorld.js

```diff
@@ -4,7 +4,8 @@
 // export default async function [name of the function called by Noteplan]
 // Type checking reference: https://flow.org/
 // Specific how-to re: Noteplan: https://github.com/NotePlan/plugins/blob/main/Flow_Guide.md
-export async function helloWorld(): Promise<void> {
+
+export const helloWorld = async (): Promise<void> => {
```

Comment:
This will be the only piece I would reject as most all other plugins in this repository use the original signature `export async`, changing to `export const` would introduce more friction.  

This can be left to the plugin developer as to the method they are most comfortable with.

Reply:
ok

Reply:
Not a comment, but a JS question: what is the benefit of the `export const ... async ...` style? I would certainly not fully understand this style.

Reply:
@jgclark have a look at this article, it breaks it down quite well.

[Variable Declaration vs Function Express](https://medium.com/@mandeep1012/function-declarations-vs-function-expressions-b43646042052)

FWIW, I would often side with using variable declaration, vs function expression, but like I mentioned to @m1well almost all of the plugins, helpers, et. Use function declaration, thus I was more look for this item to be consistent with what we have.

@m1well In general, I too would use variable declaration, but again, wanting to stay consistent :-)

Reply:
At the risk of kicking off a religion debate...it's mostly stylistic and doesn't matter, but...IMO, it would be nice to have some kind of standardization in the codebase stylistically -- especially when we're sharing helper code. Always bugs me to see a mix of styles in one code file. 

I don't do this for a job like @mikeerickson or @m1well, so take this with a grain of salt, but I'm a huge fan of Airbnb opensourcing their lint rules to that end -- keep everyone writing the same style of code. I personally tend to favor:
a) variable+anonymous functions for one-liners (e.g. no curly braces needed). If I need a curly brace, I turn it into a function declaration.
b) function declarations for all else
My experience in debugging/stack tracing is that function-name inference on variable functions is inconsistent, and it's easier to see how you got here through named functions.

Besides, it's way easier (for me) when scanning through code to see that something's a function versus someone setting a variable.

Sounds like "your mileage may vary" in opinions on this issue. So, let the style war comments begin!